07/index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="container 1"></div>
+    
+</body>
+</html>
```

Comment:
in the end of all file you have to leave a new empty line
![image](https://user-images.githubusercontent.com/32619543/78547314-92219500-7807-11ea-8bbf-77af0c97bf24.png)